submissions/jeffersonMormah/database/transport.sql

```diff
@@ -0,0 +1,39 @@
+CREATE DATABASE ohonbatransport;
+
+use ohonbatransport;
+SHOW tables;
+
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    firstname VARCHAR(50) NOT NULL,
+    lastname VARCHAR(50) NOT NULL,
+    sex VARCHAR(7) NOT NULL,
+    age INT,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE trips (
```

Comment:
Something is wrong with this table. Kindly rerun all your query (i.e from the creation of the database to the last)  you will notice the error on your own